src/main/java/Mailserver.java

```diff
@@ -0,0 +1,147 @@
+public class Mailserver {
+    String host;
+    int port;
+    boolean debug;
+    String username;
+    String password;
+    String sendermail;
+
+
+    public Mailserver() {
+        this.host = host;
+        this.port = port;
+        debug= true;
+        this.username = username;
+        this.password = password;
+        this.sendermail = sendermail;
+    }
+
+    /**
+     * Sets default mailserver to gmails SMTP:
+     */
+    public void useGmailSMTP(){
+        this.setNewMailServer("smtp.gmail.com", 465, "dd2480.lab2.group22@gmail.com", "Group_22","dd2480.lab2.group22@gmail.com" );
+    }
+
+    /**
+     * Sets default mailserver to mailtrap SMTP.
+     */
+    public void useMailTrap(){
+        this.setNewMailServer( "smtp.mailtrap.io", 2525, "6189bbfe46e3f3", "7fcb468baea694","CI-Server@kth.se" );
+    }
+
+    /**
+     *
+     * @param host
+     * @param port
+     * @param username
+     * @param password
+     * @param senderEmail
+     */
```

Comment:
Some javadoc comments are empty in the PR

Reply:
?

Reply:
Added more comments.